### PR TITLE
Documentation update and howto

### PR DESCRIPTION
**Pull request by splattner**
Documentation update with more details on the parameters
How-To for the AWS Storage Provider Plugin

## Checklist

- [x] Keep pull requests small so they can be easily reviewed.
- [x] Update the documentation.
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog

### CODE REVIEW

File: docs/modules/ROOT/pages/how-tos/aws.adoc [hunk 1]
```diff
@@ -0,0 +1,54 @@
+= Provider AWS
+
+This guide covers how to use the aws storage provider provider
```
Comment (megian) [inline docs/modules/ROOT/pages/how-tos/aws.adoc]:
```suggestion
This guide covers how to use the AWS storage provider provider.
```

File: docs/modules/ROOT/pages/how-tos/aws.adoc [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+== Use AWS  Storage Provider
+
+See https://github.com/vmware-tanzu/velero-plugin-for-aws#setup and https://github.com/vmware-tanzu/velero-plugin-for-aws for more details on the AWS Plugin.
+
+1. Set `helm_values.configuration.provicer` parameter to `aws`
```
Comment (megian) [inline docs/modules/ROOT/pages/how-tos/aws.adoc]:
```suggestion
. Set `helm_values.configuration.provicer` parameter to `aws`.
```

File: docs/modules/ROOT/pages/how-tos/aws.adoc [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+2. Add your BackupStorageLocations, VolumeSnapshotLocation and the required Secrets
```
Comment (megian) [inline docs/modules/ROOT/pages/how-tos/aws.adoc]:
```suggestion
. Add your BackupStorageLocations, VolumeSnapshotLocation and the required Secrets.
```

File: docs/modules/ROOT/pages/how-tos/aws.adoc [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+secrets:
+  aws:
+    cloud: |
+        [default]
+        aws_access_key_id =
+        aws_secret_access_key =
+
+----
+
+3. Configure your Schedules
```
Comment (megian) [inline docs/modules/ROOT/pages/how-tos/aws.adoc]:
```suggestion
. Configure your Schedules
```

File: docs/modules/ROOT/pages/how-tos/aws.adoc [hunk 5]
```diff
@@ -0,0 +1,54 @@
+= Provider AWS
+
+This guide covers how to use the aws storage provider provider
+
+== Use AWS  Storage Provider
```
Comment (megian) [inline docs/modules/ROOT/pages/how-tos/aws.adoc]:
```suggestion
== Use AWS Storage Provider
```

File: docs/modules/ROOT/pages/how-tos/aws.adoc [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+3. Configure your Schedules
+
+[source,yaml]
+----
+schedules:
+- name: full-backup
+  spec:
+    schedule: '0 7 * * *'
+    ttl: 24h0m0s
+----
```
Comment (megian) [inline docs/modules/ROOT/pages/how-tos/aws.adoc:58]: It requires a reference in the `partials/nav.adoc` file.

File: docs/modules/ROOT/pages/references/parameters.adoc [hunk 7]
```diff
@@ -18,9 +18,79 @@ default:: see `defaults.yml`
...
 All helm_values are passed to the helm chart.
 
-== Example
+== `schedules`
+
+[horizontal]
+type:: dict
+default:: []`
+
+Velero [Schedules](https://velero.io/docs/main/api-types/schedule/) to deploy. The array contains an object with the `name` and the `spec` section for the Schedule
```
Comment (megian) [inline docs/modules/ROOT/pages/references/parameters.adoc]:
```suggestion
See Velero [Schedules](https://velero.io/docs/main/api-types/schedule/) to deploy.
The array contains an object with the `name` and the `spec` section for the Schedule.
```

File: docs/modules/ROOT/pages/references/parameters.adoc [hunk 8]
```diff
@@ -18,9 +18,79 @@ default:: see `defaults.yml`
...
+
+== `backupstoragelocations`
+
+[horizontal]
+type:: dict
+default:: []`
+
+Velero [BackupStorageLocation](https://velero.io/docs/main/api-types/backupstoragelocation/) to deploy. The array contains an object with the `name` and the `spec` section for the BackupStorageLocation
```
Comment (megian) [inline docs/modules/ROOT/pages/references/parameters.adoc]:
```suggestion
See Velero [BackupStorageLocation](https://velero.io/docs/main/api-types/backupstoragelocation/) to deploy.
The array contains an object with the `name` and the `spec` section for the BackupStorageLocation.
```

File: docs/modules/ROOT/pages/references/parameters.adoc [hunk 9]
```diff
@@ -18,9 +18,79 @@ default:: see `defaults.yml`
...
+
+== `volumesnapshotlocations`
+
+[horizontal]
+type:: dict
+default:: []`
+
+Velero [VolumeSnapshotLocation](https://velero.io/docs/main/api-types/volumesnapshotlocation/) to deploy. The array contains an object with the `name` and the `spec` section for the VolumeSnapshotLocation
```
Comment (megian) [inline docs/modules/ROOT/pages/references/parameters.adoc]:
```suggestion
See Velero [VolumeSnapshotLocation](https://velero.io/docs/main/api-types/volumesnapshotlocation/) to deploy.
The array contains an object with the `name` and the `spec` section for the VolumeSnapshotLocation.
```

File: docs/modules/ROOT/pages/references/parameters.adoc [hunk 10]
```diff
@@ -18,9 +18,79 @@ default:: see `defaults.yml`
...
+
+== `secrets`
+
+[horizontal]
+type:: dict
+default:: []`
+
+Additional Secrets to deploy used by a Storage Provider Plugin (for example aws)
```
Comment (megian) [inline docs/modules/ROOT/pages/references/parameters.adoc]:
```suggestion
Additional Secrets to deploy used by a Storage Provider Plugin (for example aws).
```

File: docs/modules/ROOT/pages/references/parameters.adoc [hunk 11]
```diff
@@ -18,9 +18,79 @@ default:: see `defaults.yml`
...
+        region: 'us-west-2'
+        profile: 'default'
+
+secrets:
+  aws:
+    cloud: |
+        [default]
+        aws_access_key_id =
+        aws_secret_access_key =
+
```
Comment (megian) [inline docs/modules/ROOT/pages/references/parameters.adoc]:
```suggestion
parameters:
  velero:
    helm_values:
      configuration:
        provider: aws

    schedules:
    - name: full-backup
      spec:
        schedule: '0 7 * * *'
        ttl: 24h0m0s

    backupstoragelocations:
    - name: default
      spec:
        provider: aws
        objectStorage:
          bucket: myBucket
        credential:
          name: aws
          key: cloud
        config:
          region: 'us-west-2'
          profile: 'default'

    volumesnapshotlocations:
      - name: default
        spec:
          provider: aws
          config:
            region: 'us-west-2'
            profile: 'default'

    secrets:
      aws:
        cloud: |
            [default]
            aws_access_key_id =
```

File: docs/modules/ROOT/pages/how-tos/aws.adoc (hunk 6 above)
Comment (splattner) [reply docs/modules/ROOT/pages/how-tos/aws.adoc:58]: :+1:

**Review by megian — APPROVED** [review]

**Pull request merged**